src/nifgen/system_tests/test_system_nifgen.py

```diff
@@ -88,6 +88,40 @@ def test_self_cal(session):
     session.self_cal()
 
 
+def test_channels_rep_cap():
+    with nifgen.Session('', '', False, 'Simulate=1, DriverSetup=Model:5433 (2CH);BoardType:PXIe') as session:
+        session.func_amplitude = 0.5
+        assert session.channels['0-1'].func_amplitude == 0.5
+
+        session.channels[0].func_amplitude = 1
+        assert session.channels['0'].func_amplitude == 1
+        assert session.channels[1].func_amplitude == 0.5
+
+
+def test_markers_rep_cap(session):
+    assert '' == session.markers['Marker0'].marker_event_output_terminal
+
+    requested_terminal_name = '/Dev1/PXI_Trig0'
+    session.markers['Marker0'].marker_event_output_terminal = requested_terminal_name
+    assert requested_terminal_name == session.markers[0].marker_event_output_terminal
+
+
+def test_data_markers_rep_cap(session):
+    assert nifgen.DataMarkerEventLevelPolarity.HIGH == session.data_markers['DataMarker0'].data_marker_event_level_polarity
+
+    requested_polarity = nifgen.DataMarkerEventLevelPolarity.LOW
+    session.data_markers['DataMarker0'].data_marker_event_level_polarity = requested_polarity
+    assert requested_polarity == session.data_markers[0].data_marker_event_level_polarity
```

Comment:
I originally wrote this test to use `data_marker_event_output_terminal` but reading back the value after setting it to '/Dev1/PXI_Trig0' returned 
```
>           raise DriverError(code, description)
E           nifgen.errors.DriverError: -1074135024: IVI: (Hex 0xBFFA0010) Invalid value for parameter or property.
E
E           Property: Data Marker Event Output Terminal
E           Invalid Value: /Dev1/PXI_Trig0
E           Valid Values:

C:\Program Files\Python36\lib\site-packages\nifgen\errors.py:102: DriverError
```

Reply:
It's possible that Munich doesn't support data markers. Can you do a quick experiment with a 5421 on your end?

Keep the system test as-is.

Reply:
I think we should avoid using codenames in public forums.
I'll check the 5421 behavior and get back to you.

Reply:
I didn't have access to a 5421 but I tried to exporting the data marker event with a simulated 5421 and a real PXI-5441. Neither one errored.

I also looked through the NI-FGEN documentation and found [this page](https://zone.ni.com/reference/en-XX/help/370524V-01/pxie-5433/routes/) which seems to confirm that the PXIe-5433 (2CH) that we simulate with by default, does not support Data Marker Events. Marker Events are listed, but Data Marker Events are not.

I'll leave the test code basically as is, since you've already reviewed and okayed it, but I'm going to change this test to simulate with the 5421, instead. I don't want us to rely on the behavior for a property related to an unsupported event remaining unchanged.

Reply:
> I'll leave the test code basically as is, since you've already reviewed and okayed it, but I'm going to change this test to simulate with the 5421, instead.

Please don't do this, as simulating DAQmx based instruments has its own set of problems* and we're aggressively moving away from it in order to simplify things and speed up testing even if it means a bit worse test coverage.

I'd rather stick to 54x3 instruments. We could look at the error. Unfortunately it does not appear to tell you _which_ Data Marker Event is being configured.